src/test/utils/EigenLayerUnitTestSetup.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol";
+import "src/contracts/strategies/StrategyBase.sol";
+import "src/test/mocks/StrategyManagerMock.sol";
+import "src/test/mocks/DelegationManagerMock.sol";
+import "src/test/mocks/SlasherMock.sol";
+import "src/test/mocks/EigenPodManagerMock.sol";
+import "src/test/mocks/StakeRegistryMock.sol";
+import "src/test/utils/Utils.sol";
+import "src/test/utils/EigenLayerUnitTestBase.sol";
+
+abstract contract EigenLayerUnitTestSetup is EigenLayerUnitTestBase, Utils {
+    // Declare Mocks
+    StrategyManagerMock strategyManagerMock;
+    DelegationManagerMock public delegationManagerMock;
+    SlasherMock public slasherMock;
+    EigenPodManagerMock public eigenPodManagerMock;
+    StakeRegistryMock stakeRegistryMock;
+    // strategies
+    IERC20 public weth;
+    IERC20 public eigenToken;
+    StrategyBase public wethStrat;
+    StrategyBase public eigenStrat;
+    StrategyBase public baseStrategyImplementation;
+    uint256 wethInitialSupply = 10e50;
+    uint256 public constant eigenTotalSupply = 1e50;
+
+    function setUp() public virtual override {
```

Comment:
Should the pauser registry also be initialized here?

Reply:
`EigenLayerUnitTestSetup` is inheriting `EigenLayerUnitTestBase` which is setting up the pauser registry